python/ray/serve/api.py

```diff
@@ -126,10 +127,10 @@ def init(blocking=False,
 
     master_actor = ServeMaster.options(
         detached=True,
-        name=SERVE_MASTER_NAME,
+        name=master_actor_name,
         max_restarts=-1,
-    ).remote(queueing_policy.value, policy_kwargs, start_server, http_host,
-             http_port, metric_exporter)
+    ).remote(cluster_name, queueing_policy.value, policy_kwargs, start_server,
+             http_host, http_port, metric_exporter)
 
```

Comment:
can we check the port is available somewhere here? With multiple serve cluster, it will be quite easy to forget to assign new port number and have a port collision. Currently, we just silently failed to start HTTP actor and timeout in trying to reach HTTP, which is not a pleasant user experience